docs/dev/building.md

```diff
@@ -22,37 +22,45 @@ git clone https://github.com/scylladb/scylla
 cd ./scylla
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py --mode=<mode>
-./tools/toolchain/dbuild ninja
+./tools/toolchain/dbuild cmake --build build
 ```
 
 where `mode` is `dev` for development builds, `release` for release builds, and `debug` for debug builds.
 
-Alternatively, you can also pass the `mode` directly to the `ninja` command:
+Alternatively, you can also pass the `config` directly to the `cmake` command:
 
 ```console
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py
-./tools/toolchain/dbuild ninja <mode>
+./tools/toolchain/dbuild cmake --build build --config <config>
 ```
 
-After the build completes, you can find build artifacts in `build/<mode>` directory.
+Instead of using `cmake`, you can use `ninja` directly with the notation of
+`<target>:<config>`:
+
+```console
+./tools/toolchain/dbuild ninja -C build tests:<config>
+
+```
+
+After the build completes, you can find build artifacts in `build/<config>` directory.
 
 You can run unit tests with:
 
 ```console
-./tools/toolchain/dbuild ninja test
+./tools/toolchain/dbuild ninja -C build test
```

Comment:
I think this instruction is probably fine; however, the leading text -- **run** unit tests -- seems wrong. This command is supposed to *build* unit tests, right?

Reply:
both the building systems generated by `configure.py` and by `cmake` have a target named `test`, which actually runs `test.py`. so i think this command does run unit tests. to build unit tests, we have another target named `tests`.

Reply:
see https://github.com/scylladb/scylladb/blob/298cafff3537950f1b17b1fcf119e3d831bcd786/configure.py#L2051-L2052 and https://github.com/scylladb/scylladb/blob/298cafff3537950f1b17b1fcf119e3d831bcd786/test/CMakeLists.txt#L118